configure.ac

```diff
@@ -207,7 +207,8 @@ AS_IF([test x"$enable_hardening" != x"no"], [
   AC_DEFUN([add_hardened_c_flag], [
     AX_CHECK_COMPILE_FLAG([$1],
       [EXTRA_CFLAGS="$EXTRA_CFLAGS $1"],
-      [AC_MSG_ERROR([Cannot enable $1, consider configuring with --disable-hardening])]
+      [AC_MSG_ERROR([Cannot enable $1, consider configuring with --disable-hardening])],
```

Comment:
Something else is still broken:
```
configure: WARNING: Compiling with --disable-hardening is dangerous!
you should consider fixing the configure script compiler flags
and submitting patches upstream!
checking whether C compiler accepts -D_GNU_SOURCE... yes
checking whether C compiler accepts -Wstringop-overflow=4... no
checking whether C compiler accepts -Wstringop-truncation... no
checking whether C compiler accepts -Wduplicated-branches... no
checking whether C compiler accepts -Wduplicated-cond... no
checking whether C compiler accepts -Wbool-compare... no
configure: error: Cannot enable -Wbool-compare, consider configuring with --disable-hardening
The command "./.ci/travis.run" exited with 1.
```
Perhaps --disable-hardening configure flag is not working?

Reply:
See my comment on line 211